packages/template-chakra-storefront/refactoring-checklist.md

```diff
@@ -0,0 +1,237 @@
+# Refactoring Checklist
```

Comment:
Are we getting this in?

Reply:
No we are not committing these, i used them to automate the changes, I removed them, and then Vincent M found out some missing pieces, so i'm using AI to fix those missing pieces again.

Reply:
will remove after things are all cleaned up